ui/pages/smart-transactions/smart-transaction-status-page/smart-transaction-status-page.tsx

```diff
@@ -302,6 +302,29 @@ const CloseExtensionButton = ({
   );
 };
 
+const FooterText = ({
+  isDapp,
+  isSmartTransactionPending,
+}: {
+  isDapp: boolean;
+  isSmartTransactionPending: boolean;
+}) => {
+  if (!isDapp || !isSmartTransactionPending) {
+    return null;
+  }
+  const t = useI18nContext();
```

Comment:
Just a heads-up, it’s best to place hooks at the top of the render to avoid calling them conditionally like this.